Frontend/src/pages/DashboardPage/DashboardPage.module.css

```
@@ -0,0 +1,14 @@
+.container { }
```

Stylelint (17.10.0)

[error] Empty block (block-no-empty)

(block-no-empty)